client/js/autocompletion.js

```diff
@@ -318,7 +318,7 @@ function completeChans(word) {
 		.each(function() {
 			const self = $(this);
 
-			if (!self.hasClass("lobby")) {
+			if (self.hasClass("channel")) {
```

Comment:
This disables from queries as well, right? Is that intentional?

Reply:
So this should be `if (self.hasClass("channel") || self.hasClass("query")) {` I think.

Reply:
But what reason do you have to complete queries after typing `#` somewhere?

Reply:
Oh wait, I think I'm reading this incorrectly.

Reply:
Yes for some reason I read this as the location it's autocompleted from (which window) and not what is being populated, my bad.